tests/tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+run_test() {
+    echo -ne "Running test \\e[33m\"$(echo "$1" | sed -E "s/.*\\/(.*)\\//\\1/")\"\\e[0m... "
+
+    expected_status_code="$(perl -p -e 'BEGIN{undef $/;} s/^--STATUS-CODE--\n(\d+)\n.*$/$1/s' "$1test")"
+    expected_output="$(echo -e "$(perl -p -e 'BEGIN{undef $/;} s/^.*--OUTPUT--\n(.*)\n$/$1/s' "$1test")")"
```

Comment:
```diff
-$1test
+${1}test
```
perhaps ?

it took me a while to figure out that `test` is a filename inside `$1` dir

Reply:
Done.